### DISCUSSION TITLE
Plans for support / migration to Next.js 13

### DISCUSSION THREAD

**Discussion by fcisio** [nrwl/nx#12818] [Feature Requests]
Hi, I just wanted to open the discussion about Next.js 13 which was released just now.

1. Is there a timeline for support?
2. Will there be significant migration or we should already be able to upgrade?

It's still really fresh, so any information at all would be appreciated,
Thanks.

**Comment by ziadsabillah** [comment]
Any updates on this?

**Reply by rexfordessilfie** [reply]
I just attempted the migration and it looks to be supported now. Try running `nx migrate` command from [here](https://nx.dev/packages/nx/documents/migrate). 
For me it updated all packages, including Next.js to v13.1.1, and then I manually upgraded to v13.3.0 to be able to use new Next.js features like Google fonts.

**Comment by jaysoo** [comment]
Closing this as it is complete.
